applications/matter_weather_station/README.rst

```diff
@@ -13,7 +13,8 @@ The device works as a Matter accessory device, meaning it can be paired and cont
 You can use this application as a reference for creating your own application.
 
 .. note::
-    The Matter protocol is in an early development stage and must be treated as an :ref:`experimental <software_maturity>` feature.
+   The Thingy:53 Matter weather station application featured in the |NCS| v2.1.1 release participated in Matter Specification Validation Event (SVE) and successfully passed all required test cases to be considered as a device compliant with Matter 1.0.
+   You can use |NCS| v2.1.1 release to see the configuration and files that were originally used in Matter 1.0 certification.
```

Comment:
We need to add v2.1.1 link here when the release is live. So I suggest you remove the whole note, and I add it with the PR that adds v2.1.1 to main.

Reply:
It was removed.